buildenv/jenkins/variables/defaults.yml

```diff
@@ -431,6 +431,47 @@ x86-64_windows:
       12: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin:/cygdrive/c/openjdk/nasm-2.13.03'
       next: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin:/cygdrive/c/openjdk/nasm-2.13.03'
 #========================================#
+# Windows x86 64bits Large Heap
+#========================================#
+x86-64_windows:
+  boot_jdk:
+    8: '/cygdrive/c/openjdk/jdk7'
+    9: '/cygdrive/c/openjdk/jdk8'
```

Comment:
No sense in adding for releases that won't work or be supported.

Reply:
Good idea. Should probably follow up with a PR to remove 9/10 completely @pshipton ? Maybe adopt some kind of n-2 rule of thumb?

Reply:
Yes, 9 and 10 can be removed for all plats, but this should be a separate commit. I'm fine with not included 9 and 10 for this new platform, or it can be removed separately with everything else, I don't care.